extractor/src/main/java/org/schabi/newpipe/extractor/linkhandler/ChannelTabs.java

```diff
@@ -0,0 +1,12 @@
+package org.schabi.newpipe.extractor.linkhandler;
+
+public final class ChannelTabs {
+    public static final String SHORTS = "shorts";
+    public static final String LIVESTREAMS = "livestreams";
+    public static final String CHANNELS = "channels";
+    public static final String PLAYLISTS = "playlists";
+    public static final String ALBUMS = "albums";
+
+    private ChannelTabs() {
+    }
+}
```

Comment:
I used an enum in the initial version, but decided to use the existing LinkHandlers which take strings for content filtering. The search function works the same.

Reply:
Can't you use an enum and call `.name()` on it?

Reply:
Yes, that is an option. But I dont see an advantage in that. We wont get the type safety of an enum and it would make the code longer.

There arent any enums used for search filters, either.

Reply:
Well, sure, but we can check that they're valid by iterating over `Enum.values()`. It's more to keep the code than actual type safety clean I would say.

Reply:
Mmmh, I agree with @FireMasterK that `Enum.values()` can be useful